deploy/k8s/charts/trustification/templates/helpers/_oidc.tpl

```diff
@@ -38,7 +38,11 @@ Arguments (dict):
   {{- include "trustification.common.envVarValue" $client.clientSecret | nindent 2 }}
 
 - name: OIDC_PROVIDER_ISSUER_URL
+  {{- if .root.Values.ssoServiceName }}
```

Comment:
Why isn't it just possible to use the `.oidc.issuerUrl` variable in the values file?